anti-analysis/anti-vm/vm-detection/reference-anti-vm-strings---wmi-motherboard-model-contains-virtual.yml

```diff
@@ -0,0 +1,19 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: reference anti-VM strings - WMI motherboard model contains Virtual
```

Comment:
perhaps: "detect VM via hardware motherboard WMI queries"?

Reply:
I am good with changing the title to this, was just trying to reuse the format of the other ones that reference strings.